docs/usage.md

```diff
@@ -76,6 +76,11 @@ exemptions:
       - hostNetworkSet
 ```
 
+To exempt a controller from a particular check via annotations, use an annotation in the form of `polaris.fairwinds.com/<check>-exempt=true`, e.g.
+```
+kubectl annotate deployment my-deployment polaris.fairwinds.com/cpu-requests-missing-exempt=true
```

Comment:
Any reason not to use camelCase like we do for the IDs? From what I can tell `[A-Z]` is legal.

e.g. `cpuRequestsMissing-exempt`

Reply:
Well that'll teach me to make assumptions. I don't know why I had it in my head that annotations had to follow the same rules as namespaces or other objects. I'll change it up.